build/circle-deps.sh

```diff
@@ -36,6 +36,9 @@ function fetch_docker() {
 # the argument causing the commands in the if-branch to be executed
 # within the docker container.
 if [ "${1-}" = "docker" ]; then
+  go get -u github.com/robfig/glock
+  glock sync -n < GLOCKFILE
```

Comment:
was there a particular reason for skipping the commands? Do they take up time on every invocation now?
Hmm. looks like they're cached below.